tests/dispatch/utils.py

```diff
@@ -189,13 +189,13 @@ def __getattr__(self, name: str) -> typing.Any:
         return getattr(self.backend, name)
 
 
-class MockRawSocketStream(BaseTCPStream):
+class MockRawSocketStream(BaseSocketStream):
```

Comment:
Us already using the term “socket stream” in this mock seems like a good indication that renaming is the right thing to do. :)